core/sqf/sqenvcom.sh

```diff
@@ -505,23 +505,6 @@ EOF
   APACHE_HIVE_HOME=$HIVE_HOME
   export HIVE_CNF_DIR=$HIVE_HOME/conf
 
```

Comment:
Agree. It is good to remove this code, it proved not to be useful. I had the idea at some time to figure out the install directories for Hadoop, HBase etc. by examining the CLASSPATH, but that method caused more trouble than good. (in part because of the use of symbolic links, I think).